core/services.go

```diff
@@ -70,7 +97,12 @@ func (s *Services) Start(job models.Job) error {
 			}
 		}
 
-		ocr2, err := services.NewOCR2(job, s.db, s.webhook, s.cfg, s.keys, s.blockchain, s.peerWrapper)
+		telemetryClient, err := s.telemetryService.Start()
```

Comment:
This will still start a new telemetry per OCR job, as this service is started per OCR job.

Better to extract it here than where it was before, but we'd need another layer, a `relay.App` that is started just once, and where all root level services are started once (telemetry included).

We'll add that, no need to worry now about it.

Just as a suggestion add a TODO comment:

```go
// TODO: telemetry service should be started once (at app start), and client injected into OCR service to be used by every job. Currently, here we start a new telemetry service for every OCR job (N background connections).
telemetryClient, err := s.telemetryService.Start()
```